source/server/getting-started/implementations/spongeforge.rst

```diff
@@ -50,6 +50,80 @@ The file name ``spongeforge-1.8-1577-3.0.0-BETA-1000.jar`` is compatible with mi
     most mods shouldn't be touching, let alone modifying as Sponge does. Since Forge is free to change internal
     code whenever they want to, its normal guarantee of backwards-compatibility doesn't apply to SpongeForge.
 
+
+Installing SpongeForge
+======================
+
+.. note::
+
+    If you use (or are planning to use) a game server host, they may have a control panel that can install Sponge for you.
+
+
+.. warning::
+  When using the Mojang installer, Mojang makes use of their own Java version and not the one you installed on your
+  system. The installer currently ships with Java ``1.8.0_25`` for Windows and ``1.8.0_60`` for OSX. Note that Sponge
+  requires **at least** ``1.8.0_40`` or above to run properly. You can grab the Launcher without included Java here:
+  `official Minecraft Launcher <https://minecraft.net/download>`_
+
+Single Player / In-Game LAN Servers
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+1. Download the Minecraft Forge installer from the `Minecraft Forge website <http://files.minecraftforge.net/>`_. Make
+   sure to use **exactly the same build number** as :doc:`the one listed in the filename of the Sponge download
+   <spongeforge>`.
+#. Run the provided Forge installer. A new Forge profile will be created in the Minecraft launcher.
+#. Open the Minecraft launcher, and select the new Forge profile.
+#. Click "Options" and click "Open Game Dir".
+#. Download SpongeForge from the Sponge website and put it into the ``mods`` folder. Create the folder if it does
+   not yet exist.
+#. Sponge should work in both in single player and if you open your world to LAN.
+
+Next, learn how you can create and use a :doc:`launch-script <../launch-script>`, how to
```

Comment:
A launch script for a single-player world? Correct me if I'm wrong, but I believe that something like this belongs under Dedicated Servers.